OpenQA/Isotovideo/CommandHandler.pm

```diff
@@ -366,4 +379,26 @@ sub _handle_command_send_clients {
     $self->_respond_ok();
 }
 
+sub update_status_file {
+    my ($self) = @_;
+
+    my $coder = Cpanel::JSON::XS->new->pretty->canonical;
+
+    # ignore (openQA might not be the newest version)
+    return unless -f AUTOINST_STATUSFILE;
```

Comment:
But it wouldn't hurt to write that file, right? If so, I would actually save that condition.

This check also implies that the file is supposed to be created on the worker-side first. Wouldn't it make more sense to create the file here (instead of spreading writes to that file across multiple repos)? E.g. the file could be written in the CommandHandler's constructor. Of course the openQA worker then needs to deal with the case that the file does not exists and assume some defaults until them. (It already assumes defaults until the first update of the file on the os-autoinst side anyways.)

Reply:
I think when I started this @kraih and I agreed to create the file in the worker first. But I guess this is not necessary anymore, since the worker doesn't write any information into it.
So I'll remove the first write in the worker and remove the check here.

Reply:
done